k8s/cluster.yaml

```diff
@@ -129,7 +129,7 @@ spec:
       net.ipv4.neigh.default.gc_thresh3 = 32768
       EOT
   image: kope.io/k8s-1.17-debian-stretch-amd64-hvm-ebs-2020-01-17
-  machineType: c5.2xlarge
+  machineType: c5.4xlarge
```

Comment:
Does this instance class apply to all instances? Why are we bumping it here?

Reply:
This applies only to the `master`. Scheduling is faster with this instance, and this is the suggested instance type when running larger clusters.

We really should have this configurable in a future PR, based on the number of workers you plan to use. See https://kubernetes.io/docs/setup/best-practices/cluster-large/